web-src/src/components/PromptEditor.js

```diff
@@ -196,6 +227,22 @@ function PromptEditor({ isOpen, onClose, ...props }) {
               readOnly={!viewSource}
             />
           </div>
+
+          <Flex gap="size-100" UNSAFE_className={editorSyntaxError ? style.hidden : style.validHelpText }>
+            <CheckmarkCircle aria-label="Positive Alert" color="positive" />
+            <Text>
+              Prompt is valid.
```

Comment:
Usually, error messages are displayed only. Do we need to display this? CC: @lenaorobei